pkg/topology/topology_test.go

```diff
@@ -373,10 +373,10 @@ func TestGetTopologyInfo(t *testing.T) {
 			expectedErr: true,
 		},
 		{
-			name:        "incorrect numa node ID",
+			name:        "valid: missing numa node ID",
```

Comment:
this feels incorrect that some garbage input doesn't trigger errors. Should we distinguish between empty and wrong input?

Reply:
AFAICS wrong input gets detected earlier (the file is not found). this test case runs with a valid file but it returns no numa hints

Reply:
I agree with mythi here. As a bit of background we are seeing a lot of repeating WARNING: prints as a result of this. When a kernel is configured without NUMA support, it happens all the  time, and for those kernels which trigger the "-1 workaround" in topology.go, the NUMAs can get emptied by the workaround. End result being, at the moment every node in our cluster prints the warning repeatedly, if they happen to have a GPU in the first place.

Missing NUMAs is a rather normal situation really, which roots from kernel issues one way or the other. Not an error as such.

Reply:
thank you for explanations. LTGM